src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToGenerateImplicitNamespaceImports_DotNet.cs

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantToGenerateImplicitNamespaceImports_DotNet : SdkTest
+    {
+        private const string _testProjectName = "TestConsoleApp";
+        private const string _importFileName = "ConsoleApp.ImplicitNamespaceImports.cs";
+
+        public GivenThatWeWantToGenerateImplicitNamespaceImports_DotNet(ITestOutputHelper log) : base(log) { }
+
+        [Fact]
+        public void It_generates_imports_and_builds_successfully()
+        {
+            var tfm = "net6.0";
+            var testAsset = _testAssetsManager
+                .CopyTestAsset(_testProjectName)
+                .WithSource()
+                .WithTargetFramework(tfm);
+
+            var buildCommand = new BuildCommand(testAsset);
+            buildCommand
+                .Execute()
+                .Should()
+                .Pass();
+
+            var outputDirectory = buildCommand.GetIntermediateDirectory(tfm);
+
+            outputDirectory.Should().HaveFile(_importFileName);
+
+            File.ReadAllText(Path.Combine(outputDirectory.FullName, _importFileName)).Should().Be(
+@"// <autogenerated />
+global using global::System;
+global using global::System.Collections.Generic;
+global using global::System.IO;
+global using global::System.Linq;
+global using global::System.Net.Http;
+global using global::System.Threading.Tasks;
+");
+        }
+
+        [Fact]
+        public void It_can_disable_dotnet_imports()
```

Comment:
Assuming we move the default `Import` items to a .props file like I suggested, we should also have a test that you can remove specific ones in the project file.